heureka/ui/src/components/issueMatches/IssueMatchesDetails.jsx

```diff
@@ -41,40 +44,77 @@ const IssuesDetails = () => {
       <Stack direction="vertical" gap="4">
         <DataGrid columns={2}>
           <DataGridRow>
-            <DataGridHeadCell>Component Name</DataGridHeadCell>
+            <DataGridHeadCell>Primary Name</DataGridHeadCell>
+
+            <DataGridCell>
+              <LoadElement elem={issue?.issue?.primaryName} />
+            </DataGridCell>
+          </DataGridRow>
+          <DataGridRow>
+            <DataGridHeadCell>Target Remediation Date</DataGridHeadCell>
 
             <DataGridCell>
               <LoadElement
-                elem={
-                  issue?.componentInstance?.componentVersion?.component?.name
-                }
+                elem={formatDate(issue?.node?.targetRemediationDate)}
               />
             </DataGridCell>
           </DataGridRow>
+          <DataGridRow>
+            <DataGridHeadCell>Status</DataGridHeadCell>
+
+            <DataGridCell>
+              <LoadElement elem={issue?.status} />
+            </DataGridCell>
+          </DataGridRow>
 
           <DataGridRow>
-            <DataGridHeadCell>CVE</DataGridHeadCell>
+            <DataGridHeadCell>Severity</DataGridHeadCell>
 
             <DataGridCell>
-              <LoadElement elem={issue?.issue?.primaryName} />
+              <LoadElement elem={issue?.severity?.value} />
```

Comment:
Maybe (if you want) use for formatting: https://github.com/cloudoperators/greenhouse-extensions/blob/452d1bb6d8fe08864226bf469b46fb5a6f30dd85/heureka/ui/src/components/shared/Helper.jsx#L41C14-L41C28   will be Medium (8.1) instead of Medium

Reply:
Also it is not working at all at my machine. Sadly i cant revoke my approval - can you check this please?

Reply:
yes, I used the helper and removed the redundant prop at the the end of the list.